### DISCUSSION TITLE
报错 状态码: 209001(未知错误码)

### DISCUSSION THREAD

**Discussion by guotaibao521** [mikumifa/biliTickerBuy#444] [提问专区]
是不是一次只能抢一个人的票 ， 选了3个人的购票信息，然后抢票的时候报错。

**Comment by mikumifa** [comment]
给具体信息，还有票id

**Comment by guotaibao521** [comment]
2024-10-16 19:00:03.851 | INFO     | tab.go:inner_request:574 - 状态码: 209001(未知错误码), 请求体: {'errno': 209001, 'errtag': 0, 'msg': '请选择1位购买人', 'data': {}}

票id=93371
![QQ20241016-191838](https://github.com/user-attachments/assets/4fec55c5-41b6-430a-9f68-62fe400a102d)

**Reply by mikumifa** [reply]
选了3个购买人应该不会出现这个问题

**Reply by guotaibao521** [reply]
程序加载正常，1个联系人，3个购买人填写也正常，不知道为何出现这种问题，

**Reply by mikumifa** [reply]
给个票id

**Reply by guotaibao521** [reply]
票id=93371

**Comment by mikumifa** [comment]
查到了，那个票好像是多个票只要一个实名认证人，以前没遇到这种情况

**Reply by guotaibao521** [reply]
确实如此 现在B站有些需要多票多实名，有些只需要一个实名即可
